scss/components/masonry-block/_masonry-block.scss

```diff
@@ -25,12 +25,6 @@
       color: color(white);
     }
 
-    .masonry-block__link {
-      @include on-event {
-        color: rgba(255, 255, 255, .65);
```

Comment:
So it is OK to not have a hover effect? Is that because it is too subtle anyhow? General thought: if we removed social media icons from cards, the whole card could have a hover state.

Reply:
Only the white titles had a hover effect and that effect was super low-contrast.